### PR TITLE
Add msw

### PR DESCRIPTION
**Pull request by evulins**
Mock Service Worker server has been added for the development environment. Setup was made based on MSW documentation -> https://mswjs.io/

### CODE REVIEW

File: src/mocks/browser.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// src/mocks/browser.js
```
Comment (franiakat) [inline src/mocks/browser.tsx]: In plain Typescript files without any JSX elements we don't need to support it, so the file extension should be `browser.tsx`. ;)

File: src/mocks/handlers.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// src/mocks/handlers.js
```
Comment (franiakat) [inline src/mocks/handlers.tsx]: In plain Typescript files without any JSX elements we don't need to support it, so the file extension should be `browser.tsx`.

File: src/mocks/handlers.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    // If authenticated, return a mocked user details
+    return res(
+      ctx.status(200),
+      ctx.json({
+        username: 'admin',
+      }),
+    )
+  }),
+]
```
Comment (franiakat) [inline src/mocks/handlers.tsx]: Empty line at the end is missing - in the future prettier will do this for you. 🙃

**Review by franiakat — CHANGES_REQUESTED** [review]
Congrats on your first PR 🍾 @evulins & @bazelajoanna 🎉 
Please see the comments.

**Comment by franiakat** [comment]
One additional thing - it is a good practice to add some description in the PR, so it can be used as a documentation in the future.
It would be helpful to say e.g. why given PR was created, or based on which sources (some links to documentation, designs - depending on the PR matter). Please, add one. :)

File: src/mocks/browser.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// src/mocks/browser.js
+import { setupWorker } from 'msw'
+import { handlers } from './handlers'
+
+// This configures a Service Worker with the given request handlers.
+export const worker = setupWorker(...handlers)
```
Comment (franiakat) [inline src/mocks/browser.js]: Please, add empty line at the end - didn't spot it before.

File: package.json [hunk 5]
```diff
@@ -40,7 +40,11 @@
...
   "devDependencies": {
+    "msw": "^0.35.0",
     "ts-loader": "^9.2.6",
     "typescript": "^4.4.4"
+  },
+  "msw": {
+    "workerDirectory": "public"
   }
-}
+}
```
Comment (franiakat) [inline package.json:50]: Please, add empty line at the end - didn't spot it before.

Comment (evulins) [reply package.json:50]: Done 👍

**Review by franiakat — APPROVED** [review]
🥂

**Pull request merged**